### PR TITLE
add repl examples for string functions and operators

### PR DESCRIPTION
**Pull request by fmoor**

### CODE REVIEW

**Comment by fmoor** [comment]
removed link to postgresql LIKE operator

File: doc/edgeql/funcop/string.rst [hunk 1]
```diff
@@ -105,4 +132,12 @@ String Functions and Operators
...
-    Return ``TRUE`` if there is a match, ``FALSE`` otherwise.
+    Return ``True`` if there is a match, ``False`` otherwise.
+
+    .. code-block:: edgeql-repl
+
+        db> SELECT std::re_test('abc', 'a');
+        {True}
+
+
+.. _SQL: https://www.postgresql.org/docs/10/static/functions-matching.html#FUNCTIONS-LIKE
```
Comment (elprans) [inline doc/edgeql/funcop/string.rst]: This is now not necessary.

Comment (fmoor) [reply doc/edgeql/funcop/string.rst]: Oops! it is actually gone now.

**Comment by fmoor** [comment]
any other changes needed here?

**Comment by elprans** [comment]
All good. Thanks for contributing!

**Pull request merged**